ios/bottom-tabs/RNSBottomTabsScreenComponentView.mm

```diff
@@ -366,9 +366,9 @@ - (void)setTabBarBackgroundColor:(UIColor *)tabBarBackgroundColor
   _tabItemNeedsAppearanceUpdate = YES;
 }
 
-- (void)setTabBarBlurEffect:(UIBlurEffect *)tabBarBlurEffect
+- (void)setTabBarBlurEffectFromRNSBlurEffectStyle:(RNSBlurEffectStyle)tabBarBlurEffect
```

Comment:
I see what you've done here. Please add a comment that this is a paper only method that will be called by the mounting code.